package.json

```diff
@@ -167,17 +167,5 @@
     "react-test-renderer": "^18.2.0",
     "set-value": "^4.1.0",
     "typescript": "~5.3.3"
-  },
-  "resolutions": {
-    "@expo/config-plugins": "^9.0.0",
-    "@babel/traverse": "~7.25.0",
-    "crypto-js": "~4.2.0",
-    "markdown-it": "~13.0.2",
-    "metro": "^0.81.0",
-    "metro-resolver": "^0.81.0",
-    "metro-config": "^0.81.0",
-    "micromatch": "~4.0.4",
-    "node-fetch": "~2.6.7",
```

Comment:
I can't try right now. I have an internet problem I don't know why. When I do a speedtest, it shows 260+ mbit.

<img width="748" alt="image" src="https://github.com/user-attachments/assets/39c10620-5a3c-4702-b0aa-687d943f3784" />

---

You didn't build a new developer, right? Or local build. You ran the app in a previously created build, right?

Reply:
Yes

Reply:
I turned my computer off and on and the problem was solved. I was able to install the packages without any problem. I also created a build locally and it works. 👍